src/sentry/static/sentry/app/views/settings/organizationMembers/organizationMembersWrapper.tsx

```diff
@@ -54,27 +54,9 @@ class OrganizationMembersWrapper extends AsyncView<Props, State> {
     return organization.access.includes('member:write');
   }
 
-  get showInviteRequests() {
-    return this.hasWriteAccess;
-  }
-
-  get showNavTabs() {
-    const {requestList} = this.state;
-
-    // show the requests tab if there are pending team requests,
-    // or if the user has access to approve or deny invite requests
-    return (requestList && requestList.length > 0) || this.showInviteRequests;
-  }
-
   get requestCount() {
     const {requestList, inviteRequests} = this.state;
-    let count = requestList.length;
-
-    // if the user can't see the invite requests panel,
-    // exclude those requests from the total count
-    if (this.showInviteRequests) {
-      count += inviteRequests.length;
-    }
```

Comment:
I'm not sure I totally understand what this logic was doing in the first place. I tested out out the team request functionality after implementing these changes, as an an superuser I can see the team requests, and as a non-superuser with member:read access, I cannot see team requests. Is that..... correct? (It seems correct?)

Reply:
yeah so owners, managers, and team admins (admins that are members of the requested team) can read and approve team access requests, which occur when open membership is disabled. i think this was removing invite requests from the count for members who couldn't view the tab. your api changes should resolve this